linalg/sparsemat.hpp

```diff
@@ -151,8 +165,8 @@ class SparseMatrix : public AbstractSparseMatrix
    /// Create a SparseMatrix with diagonal @a v, i.e. A = Diag(v)
    SparseMatrix(const Vector & v);
 
-   // Runtime option to use cuSPARSE. Only valid when using a CUDA backend.
-   void UseCuSparse(bool useCuSparse_ = true) { useCuSparse = useCuSparse_;}
+   // Runtime option to use cuSPARSE or hipSPARSE. Only valid when using a CUDA or HIP backend.
+   void UseGPUSparse(bool useGPUSparse_ = true) { useGPUSparse = useGPUSparse_;}
```

Comment:
Keep `UseCuSparse` as deprecated --  use same implementation as the new UseGPUSparse.

Reply:
done